Add header and Hero sections

jecode93 opened the pull request:
## Portfolio project

For this portfolio I :

- Structure the website
- Create a mobile-first website with a minimum mobile screen size of 375px
- Use HTML & CSS
- Use Flexbox
- Use HTML&CSS linter

jecode93 committed:
Created a structure for the website files

jecode93 committed:
Website content part 1

- Created header parts for the website
- Created hero section

jecode93 committed:
Styling the portfolio website

- Link the css file with the html file
- Style the header section
- Style the hero section
- Styling website with CSS and Flexbox for mobile first

jecode93 committed:
Updated the readme file

jecode93 committed:
Update the styles file as request by stylelint

skyv26 commented on html/index.html:
```diff
@@ -0,0 +1,37 @@
+  </head>
+
+  <body>
+    <div class="wrapper">
```
- [ ] Please follow the `HTML` best practices by using the right semantics tags, as I can see that you are using the `div` for giving design to the whole body content. It would be better if you give the CSS design directly to body or make separate class which you can add directly to the `header` and `main` section because right now even I can see that you are not giving any style to this **div** tag so please remove it.

skyv26 commented on html/index.html:
```diff
@@ -0,0 +1,37 @@
+  <body>
+    <div class="wrapper">
+      <header>
+        <h2>My Logo</h2>
```
- [ ] Please use the `a` tag instead of the `h2` tag. `hx` tags are meant for the main content **headings** or **subheadings** not for the logo, and it is semantically not correct which can result in *bad user experience*. You can take an example of a person and just assume that your website is bigger and have several pages and a user just lost somewhere and that person wants to go back at home page then in that case apart from `Home` navigation this `logo` with the help of `a` tag makes that person work easier to securely come back on `home-page`.

skyv26 commented on html/index.html:
```diff
@@ -0,0 +1,37 @@
+            <a href="#"><img src="../images/icons/socials/m.svg" alt="M icon"></a>
+            <a href="#"><img src="../images/icons/socials/github.svg" alt="GitHub icon"></a>
+            <a href="#"><img src="../images/icons/socials/hand.svg" alt="Hand icon"></a>
+          </div>
```
- [ ] Please 🙏🏻  consider using the `ul` tag as a container and put all the social media icon links in individual `li` tags, it will make your code more professional and semantically correct. It will also help **visually-impaired** people to navigate through your website with a good *User Experience*.

- [ ] Please once you change it make sure that you must have to add styles for it in the CSS and use the flexbox there to aligned the item properly.

- [OPTIONAL] Please make sure that always close your tag with the right `tagName` otherwise as your code scale or as you will add more and more features to it, later on, it will be difficult for you to find the error just because of a *small* mistake.

skyv26 commented on html/index.html:
```diff
@@ -0,0 +1,37 @@
+    <div class="wrapper">
+      <header>
+        <h2>My Logo</h2>
+        <img class="hamburger-icon" src="../images/icons/hamburger.png" alt="Menu Icon">
```
- [ ] Please wrap this image tag inside the `nav` tag because soon in the upcoming milestone you will add menus and this hamburger icon is also part of website navigation so it is better to use the `nav` tag.

skyv26 commented on html/index.html:
```diff
@@ -0,0 +1,37 @@
+          <p class="desc">I'm a software developer! I can help you build a product , feature or website Look through some of my
+            work
+            and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+          <p class="btn-connect">Lest's connect</p>
```
- [ ] Please fix the typo it is *Let's* not **Lest's**

skyv26 commented on README.md:
````diff
@@ -1,288 +1,147 @@
+- Modify the name in the h1 tag or the description
+- Refresh you browser to see the changes
 
-### Deployment
 
-You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
````
- [ ] Please add this section back because except for `FAQ` rest of everything is required and you must have to add it because the template provided by *Microverse* is fully professional. You can write something about the web platforms which provide deployment services as information in this section.

skyv26 commented on README.md:
```diff
@@ -1,288 +1,147 @@
+- **[Pure html]**
+- **[Pure css]**
 
-- [Live Demo Link](https://google.com)
```
- [ ] Please add this section back because except for `FAQ` rest of everything is required and you must have to add it because the template provided by *Microverse* is fully professional. You can give a text like "Website not deployed yet."

skyv26 reviewed (changes requested):
## Hi @jecode93, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 

![](https://media.giphy.com/media/3o7aCPbznb29hgOgG4/giphy.gif)

## Highlights
- Correct `GitHub` flow ✅ 
- Professional `README` template ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jecode93 committed:
Make changes to index.html file

- Added anchor tag for the logo
- Added nav tag for the hamburger menu icon
- Added ul tag as a container and li tag to make the code semantically correct
- Correct the button text from Lest's to Let's
- Remove div as a wrapper for the whole body content

jecode93 committed:
Make changes to the CSS file

- Styling the logo
- Styling the social icon list

jecode93 committed:
Update Readme file

- Added Live demo section
- Added Deployment section

jecode93 commented:
Hello @skyv26, 

Thank you a lot for your feedback. It was very helpful.
I did all the changes as you ask!

skyv26 commented:
> Hello @skyv26,
> 
> Thank you a lot for your feedback. It was very helpful. I did all the changes as you ask!

@jecode93 Well done, Please submit this same PR link for review. One of the code reviewer will give me you code review.

Ofarouq310 commented on README.md:
```diff
@@ -1,288 +1,161 @@
 
-> Describe your project in 1 or 2 sentences.
+# Portfolio
+
+> Portfolio Microverse Project 1
 
```
[OPTIONAL]
- [ ] Kindly add more of a descriptive meaningful description to your project, as this description may seem pretty vague.

Ofarouq310 commented on README.md:
```diff
@@ -1,288 +1,161 @@
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
 > Describe 1 - 3 features you will add to the project.
 
```
[OPTIONA] 
- [x] Kindly remove this line as it was just for your guidance, leaving it may seem unprofessional and inappropriate.

Ofarouq310 commented on README.md:
```diff
@@ -1,288 +1,161 @@
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+## 📝 License
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+This project is [MIT](./LICENSE) licensed.
```
[OPTIONAL]
- [x] Kindly add a `LICENSE` file to your project files, as this link is broken due to its absence, watch this quick [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI) on how to add one, and remember to use the `MIT` license.

Ofarouq310 reviewed (approved):
Hi @jecode93 , ✋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🥇 

![giphy](https://github.com/jecode93/portfolio/assets/102612396/0305ffdb-2095-44ec-b01b-b799a0e008dc)

## Highlights
✅ No linters errors.
✅ Descriptive PR summary.

## Optional suggestions
- I found that you did not use `Figma`'s specified font families in your project, I hereby recommend you follow the design requirements for the sake of your practice. 

- Your branch should have a name of a feature or action you are implementing, in your case, it could be "mobile-setup" or "header-and-deadline" or whatever suits you. Kindly fix it and rename your branch. Kindly refer to the next image for the steps of renaming a branch.
![image](https://github.com/Fentahunn/portfolio_one/assets/102612396/3212d6a3-479d-46d6-9bb2-e049ca9aa122)


- Your Pull request title should be short, and concise, and addresses what you are implementing, in your case, it could be, `Add header and headline sections` or something similar, simpler. Please refer to the next picture to help you how to edit the title.
![image](https://github.com/Fentahunn/portfolio_one/assets/102612396/acfbea30-fa50-4379-91f9-0bb9d45bd16d)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

jecode93 committed:
Update readme file

- Resolve license broken link
- Remove guidance text
